.github/workflows/linux_sanitizers.yml

```diff
@@ -483,6 +494,90 @@ jobs:
         uses: actions/upload-artifact@b4b15b8c7c6ac21ea08fcf65892d2ee8f75cf882 # v4.4.3
         if: ${{ !cancelled() }}
         with:
-          name: test-results-cpp_${{ matrix.SANITIZER }}
+          name: unit-test-results-cpp_${{ matrix.SANITIZER }}
+          path: ${{ env.INSTALL_TEST_DIR }}/TEST*.xml
+          if-no-files-found: 'error'
+
+  CXX_Func_Tests:
+    # todo: move another plugin's func tests here
+    name: C++ func tests
+    if: ${{ github.repository_owner == 'openvinotoolkit' }}
+    needs: [Docker, Build]
+    timeout-minutes: 120
+    runs-on: aks-linux-16-cores-32gb
+    container:
+      image: ${{ fromJSON(needs.docker.outputs.images).ov_build.ubuntu_22_04_x64 }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      max-parallel: 3
+      fail-fast: false
+      matrix:
+        include:
+          - SANITIZER: 'AddressAndLeak'
+          - SANITIZER: 'UndefinedBehavior'
+          - SANITIZER: 'Thread'
+    env:
+      INSTALL_DIR: /__w/openvino/openvino/install
+      INSTALL_TEST_DIR: /__w/openvino/openvino/install/tests
```

Comment:
move those env to the common env too, like you did for build dirs

Reply:
Applied